shared/team-building/index.tsx

```diff
@@ -132,15 +138,178 @@ const ContactsBanner = ({onRedoSearch, onRedoRecs}: {onRedoSearch: () => void; o
 }
 
 class TeamBuilding extends React.PureComponent<Props, {}> {
+  sectionListRef = React.createRef<Kb.SectionList>()
   componentDidMount = () => {
     this.props.fetchUserRecs()
   }
 
+  _alphabetIndex = () => {
+    return (
+      <Kb.Box2 direction="vertical" centerChildren={true} style={styles.alphabetIndex}>
+        {this.props.recommendations &&
+          this.props.recommendations.map(section =>
+            section.label && section.label.length <= 3 ? (
+              <React.Fragment key={section.label}>
+                <Kb.Text type="BodyTiny" onClick={() => this._onScrollToSection(section.label)}>
+                  {section.label}
+                </Kb.Text>
+                <Kb.Box style={styles.shrinkingGap} />
+              </React.Fragment>
+            ) : null
+          )}
+        {/* TODO get 0-9 section down here */}
+      </Kb.Box2>
+    )
+  }
+
+  _onScrollToSection = (label: string) => {
+    if (this.sectionListRef && this.sectionListRef.current) {
+      const ref = this.sectionListRef.current
+      const sectionIndex =
+        (this.props.recommendations &&
+          this.props.recommendations.findIndex(section => section.label === label)) ||
+        -1
+      if (sectionIndex >= 0 && Styles.isMobile) {
+        // @ts-ignore due to no RN types
+        ref.scrollToLocation({
+          itemIndex: 0,
+          sectionIndex,
+        })
+      }
+    }
+  }
+
+  _getRecLayout = (
+    sections: Array<SearchRecSection>,
+    indexInList: number
+  ): {index: number; length: number; offset: number} => {
+    let numSections = 0
+    let numData = 0
+    let length: 40 | 64 = 64
```

Comment:
What's the meaning behind these magic numbers?

Reply:
Ok, made some changes, let me know if they still don't make sense.